src/renderer/services/eth2/client/eth2ApiClient/cgEth2BeaconStateApi.ts

```diff
@@ -52,7 +53,8 @@ export class CgEth2BeaconStateApi implements ICGBeaconStateApi {
                 case: "snake",
             }) as ICGValidatorResponse;
         } catch (e) {
-            logger.error("Failed to fetch validator", {validatorId: id, error: e.message});
+            if (!e.message.includes('"code":404'))
```

Comment:
isn't that axios error? It should contain status code property

Reply:
https://github.com/axios/axios#handling-errors

Reply:
https://github.com/NodeFactoryIo/ChainGuardian/blob/23a6deb61e834bfc91b951a63260a194c5d6b014/src/renderer/services/api/http/httpClient.ts#L81

inside `HttpClient` is transformed from `AxiosError` to `Error`

Reply:
Can we modify it to be http error?

Reply:
we can, but not sure what time its gonna take to refactor everything